doc/source/user_guide/timeseries.rst

```diff
@@ -1780,6 +1777,50 @@ natural and functions similarly to :py:func:`itertools.groupby`:
 
 See :ref:`groupby.iterating-label` or :class:`Resampler.__iter__` for more.
 
+.. _timeseries.adjust-the-start-of-the-bins:
+
+Use `origin` or `offset` to adjust the start of the bins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1.0
+
+The bins of the grouping are adjusted based on the beginning of the day of the time series starting point. This works well with frequencies that are multiples of a day (like `30D`) or that divide a day evenly (like `90s` or `1min`). This can create inconsistencies with some frequencies that do not meet this criteria. To change this behavior you can specify a fixed Timestamp with the argument ``origin``.
+
+For example:
+
+.. ipython:: python
+
+    start, end = "2000-10-01 23:30:00", "2000-10-02 00:30:00"
+    middle = "2000-10-02 00:00:00"
+    rng = pd.date_range(start, end, freq="7min")
+    ts = pd.Series(np.arange(len(rng)) * 3, index=rng)
+    ts
+
+Here we can see that, when not using ``origin``, the result after "2000-10-02 00:00:00" are not identical depending on the start of time series:
+
+.. ipython:: python
+
+    ts.resample("17min").sum()
+    ts[middle:end].resample("17min").sum()
+
+
+Here we can see that, when using ``origin``, the result after "2000-10-02 00:00:00" are identical depending on the start of time series:
+
+.. ipython:: python
+
+   origin = pd.Timestamp("1970-01-01")
+   ts.resample("17min", origin=origin).sum()
+   ts[middle:end].resample("17min", origin=origin).sum()
+
+
+If needed you can just adjust the bins with an ``offset`` Timedelta that would be added to the default ``origin``.
+Those two examples are equivalent for this time series:
+
+.. ipython:: python
+
+    ts.resample("17min", origin=start).sum()
```

Comment:
same comment

Reply:
I have build the doc before committing this, here is what I see:

![timeseries_doc_3](https://user-images.githubusercontent.com/3902859/78950905-861c2880-7ad0-11ea-8f50-cb23e1c84da4.png)

Since we see the output of this line, I think I can set this conversation as Resolve safely :slightly_smiling_face:

Reply:
Thanks!